src/server/svr_jobfunc.c

```diff
@@ -7028,41 +7018,28 @@ recreate_exec_vnode(job *pjob, char *vnodelist, char *err_msg,
 						if (sel_entry > 0) {
 							/* there's already previous */
 							/* select/schedselect entry */
-							if (pbs_strcat(
-								&new_select,
-								&ns_malloced,
-								"+") == NULL) {
-								log_err(-1, id, "pbs_strcat failed");	
+							if (pbs_strcat(&new_select, &ns_malloced, "+") == NULL) {
+								log_err(-1, __func__, "pbs_strcat failed");	
 								goto recreate_exec_vnode_exit;
 							}
 						}
-						if (pbs_strcat( &new_select,
-							&ns_malloced,
-								"1") == NULL) {
-							log_err(-1, id,
+						if (pbs_strcat(&new_select, &ns_malloced, "1") == NULL) {
+							log_err(-1, __func__,
 							"pbs_strcat failed");
 							goto recreate_exec_vnode_exit;
 						}
-						if (pbs_strcat( &new_select,
-							&ns_malloced,
-								buf_sum) == NULL) {
-							log_err(-1, id,
+						if (pbs_strcat(&new_select, &ns_malloced, buf_sum) == NULL) {
+							log_err(-1, __func__,
 							"pbs_strcat failed");
```

Comment:
Combine the above two log_err() calls into one line each.  They won't be longer than the if() statement.